gradle.properties

```diff
@@ -1,6 +1,7 @@
 # Sets default memory used for gradle commands. Can be overridden by user or command line properties.
 # This is required to provide enough memory for the Minecraft decompilation process.
 org.gradle.jvmargs = -Xmx3G
+org.gradle.daemon = false
```

Comment:
Gradle runs a daemon process in the background by default so it doesn't have to load from scratch every time. This is leveraged by IDEs and I don't know if they'll care about this config, but if you run gradle from command-line it would prevent the daemon from starting as though you issued `--no-daemon` as an argument.

I use that flag when I'm compiling different projects in CLI on different versions of Gradle which will kick the old daemon out anyway, or I just otherwise don't want the daemon to hang around in RAM.